src/clone.d

```diff
@@ -367,23 +403,19 @@ extern (C++) FuncDeclaration hasIdentityOpEquals(AggregateDeclaration ad, Scope*
         for (size_t i = 0;; i++)
         {
             Type tthis = null; // dead-store to prevent spurious warning
-            if (i == 0)
-                tthis = ad.type;
-            if (i == 1)
-                tthis = ad.type.constOf();
-            if (i == 2)
-                tthis = ad.type.immutableOf();
-            if (i == 3)
-                tthis = ad.type.sharedOf();
-            if (i == 4)
-                tthis = ad.type.sharedConstOf();
-            if (i == 5)
-                break;
+            if (i == 0) tthis = ad.type;
+            if (i == 1) tthis = ad.type.constOf();
+            if (i == 2) tthis = ad.type.immutableOf();
+            if (i == 3) tthis = ad.type.sharedOf();
+            if (i == 4) tthis = ad.type.sharedConstOf();
+            if (i == 5) break;
```

Comment:
``for (size_t i = 0; i < 5; i++)`` then turn this into a ``final switch`` statement?